brewtils/rest/client.py

```diff
@@ -611,31 +612,76 @@ def get_file(self, file_id, **kwargs):
         Returns:
             Requests Response object
         """
-        return self.session.get(self.file_url + file_id, **kwargs)
+        return self.session.get(self.file_url + "?file_id=" + file_id, **kwargs)
```

Comment:
Instead of inline parameters, you can pass a parameter object.

return self.session.get(self.file_url, params={'file_id':file_id}, **kwargs)

Reply:
This was left as-is intentionally to allow for providing `params` as part of the `kwargs`.  It isn't needed now, but since this is known ahead of time I didn't see the problem with leaving it as written.